hibike/devices/BatteryBuzzer/disp_8.cpp

```diff
@@ -0,0 +1,201 @@
+//code that deals with displaying the data to the outside world.
+
+#include <SevenSeg.h>
+#include "disp_8.h"
+
+//Segment Pins, A toG then decimal pt.
+SevenSeg disp(8,9,7,2,0,15,14);
+
+const int numOfDigits=4;
+int digitPins[numOfDigits] = {6, 5, 4, 3};
+
+
+unsigned long last_LED_time = 0;  //Time the last LED switched
+int sequence = 0; //used to switch states for the display.  Remember that the hangle_8_segment cannot be blocking.
+
+int segment_8_run = 0;  //0 for the normal voltage readout.  1 for "Clear Calibration".  2 for "New Calibration"
+
+void setup_display()
+{
+  disp.setDigitPins(numOfDigits,digitPins);
+  disp.setDPPin(1);  //set the Decimal Point pin to #1   
+}
+
+void test_display() //a function to ensure that all outputs of the display are working.
```

Comment:
I should point out that this "Test_Display()" function actually doesn't work all that well.

It turns out that if there are two LED pins shorted, you'll still see "8.8.8.8." displayed.  It would be better to have a test function that lights up the display one segment at a time.

Not sure if I'd be willing to block merge into master on this, i think that this may be more of a 'next year' thing.